cypher/cypher-docs/src/docs/dev/query-tuning.asciidoc

```diff
@@ -35,20 +35,52 @@ To read more about the execution plan operators mentioned in this chapter, see <
 [[how-are-queries-executed]]
 == How are queries executed?
 
+_***XXTODO by Docs Team - change this heading to something like 'Preparser options' (this is what they are called internally), or something more reader-friendly (if a term other than 'preparser' is decided upon, please also change accordingly in the text). This section no longer deals with 'how queries are executed' per se. More of that content will be added by me later on. END TODO***_
+
+[abstract]
+--
+This section describes the preparser options available in Cypher.
+--
+
+Queries are able to be fine-tuned through the use of _preparser options_.
+If there is a requirement to use one or more of these options, the query must be prepended with `CYPHER`, followed by the preparser option(s), as exemplified thus: `CYPHER preparser-option1 [further preparser options] query`.
+
+
+[[cypher-preparser-version]]
+=== Cypher version
+
+Occasionally, there is a requirement to use a previous version of the Cypher compiler when running a query.
+Here we detail the available versions:
+
+[options="header"]
+|===
+| Preparser option | Description | Default?
+| `2.3` | This will force the query to use Neo4j Cypher 2.3.  |
+| `3.1` | This will force the query to use Neo4j Cypher 3.1. |
+| `3.2` | This will force the query to use Neo4j Cypher 3.2. As this is the default version, it is not necessary to use this option explicitly. | X
+|===
+
+
+[[cypher-preparser-planner]]
+=== Cypher planner
+
 Each query is turned into an execution plan by something called the _execution planner_.
 The execution plan tells Neo4j which operations to perform when executing the query.
-Two different execution planning strategies are included in Neo4j:
 
-Rule::
-This planner has rules that are used to produce execution plans.
-The planner considers available indexes, but does not use statistical information to guide the query compilation.
+Neo4j uses a _cost_-based execution planning strategy (known as the 'cost' planner): the statistics service in Neo4j is used to assign a cost to alternative plans and picks the cheapest one.
+
+All previous versions of Neo4j also included a rule-based planner, which used rules to produce execution plans.
```

Comment:
This could be simplified to:

All previous versions of Neo4j also included a rule-based planner, which used rules and available indexes, but did not consider statistical information, producing inferior performing execution plans, and has therefore been removed in Neo4j 3.2.

Reply:
I think the new sentence is more difficult to parse (too long, and too many commas)